src/authoring-app/hooks/use-s3.tsx

```diff
@@ -0,0 +1,227 @@
+import { useState } from "react";
+import { S3ResourceHelper, IS3ResourceHelperOpts} from "../utils/s3-resource-helper";
+import { S3Resource } from "@concord-consortium/token-service";
+import defaultExperiment from "../data/default-experiment.json";
+
+let helper:S3ResourceHelper = null as unknown as S3ResourceHelper;
```

Comment:
You can also type this as `helper: S3ResourceHelper | null` which would give you better type checking on references to `helper`.